operators/specifying-a-requirements.txt.md

```diff
@@ -18,3 +14,5 @@ def valid_sentiment_prediction(reviews: pd.DataFrame) -> pd.DataFrame:
     model = transformers.pipeline("sentiment-analysis")
     return reviews.join(pd.DataFrame(model(list(reviews["review"]))))
 ```
+
+**Note that if neither `requirements.txt` file nor function specific requirements are specified, `MissingRequirementsError` exception would be raised.
```

Comment:
Can we just move this line up to after line 3? And get rid of the `Note that` prefix.